src/routes/event.js

```diff
@@ -1,15 +1,17 @@
-import { Router } from 'express';
-import asyncHandler from '../middlewares/asyncHandler';
-import eventController from '../controllers/event';
-import { auth, adminAuth } from '../middlewares/authorization';
+import { Router } from "express";
+import asyncHandler from "../middlewares/asyncHandler";
+import eventController from "../controllers/event";
+import { auth, adminAuth } from "../middlewares/authorization";
+import { getSignedUrl } from "../controllers/files";
 
 
 const eventRouter = new Router();
 
 eventRouter
-    .post('/', [auth, adminAuth], asyncHandler(eventController.postEvent))
-    .get('/', [auth, adminAuth], asyncHandler(eventController.getEvents))
-    .delete('/:id', [auth, adminAuth], asyncHandler(eventController.deleteEvent))
-    .put('/:id', [auth, adminAuth], asyncHandler(eventController.updateEvent))
+    .post("/", [auth, adminAuth], asyncHandler(eventController.postEvent))
```

Comment:
Can we have some validations for the event, like validation for the `name`, `photoUrl`, and `caption`?

Reply:
Can we also change the `name` to `title`? I think it is self-explanatory to have a `title` of the event than having a `name` of the event.